### ISSUE TITLE
Impossible to create App Store Connect API Key (apiKey resource does not exists)

### ISSUE THREAD

**Issue by Nek-** [expo/expo-cli#4521]
### Summary

The command `expo credentials` fails to create App Store Connect API Key.

### Environment

```
$ npx expo-env-info
  expo-env-info 1.0.5 environment info:
    System:
      OS: macOS 12.5
      Shell: 5.8.1 - /bin/zsh
    Binaries:
      Node: 16.16.0 - /opt/homebrew/bin/node
      Yarn: 1.22.15 - /opt/homebrew/bin/yarn
      npm: 8.11.0 - /opt/homebrew/bin/npm
    Managers:
      CocoaPods: 1.10.1 - /usr/local/bin/pod
    SDKs:
      iOS SDK:
        Platforms: DriverKit 21.4, iOS 15.5, macOS 12.3, tvOS 15.4, watchOS 8.5
    IDEs:
      Android Studio: Chipmunk 2021.2.1 Patch 1 Chipmunk 2021.2.1 Patch 1
      Xcode: 13.4.1/13F100 - /usr/bin/xcodebuild
    npmPackages:
      @expo/webpack-config: ^0.17.0 => 0.17.2 
      expo: ^46.0.0 => 46.0.7 
      react: 18.0.0 => 18.0.0 
      react-dom: 18.0.0 => 18.0.0 
      react-native: 0.69.4 => 0.69.4 
      react-native-web: ~0.18.7 => 0.18.7 
    Expo Workflow: managed
```

### Please specify your device/emulator/simulator platform, model and version

none at this point

### Error output

```
✔ What do you want to do? › Set up your project to use an API Key for EAS Submit
✔ Generate a new App Store Connect API Key? … yes
✖ Failed to create App Store Connect API Key.
UnexpectedAppleResponse: The specified resource does not exist - There is no resource of type 'apiKeys' with id 'A3FZ49PU57'
UnexpectedAppleResponse: The specified resource does not exist - There is no resource of type 'apiKeys' with id 'A3FZ49PU57'
    at IrisClientAPI.parseResponse (/Users/me/.config/yarn/global/node_modules/eas-cli/node_modules/@expo/apple-utils/build/index.js:1:915555)
    at IrisClientAPI.requestAndParseAsync (/Users/me/.config/yarn/global/node_modules/eas-cli/node_modules/@expo/apple-utils/build/index.js:1:914109)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async IrisClientAPI.fetchSingleModelAsync (/Users/me/.config/yarn/global/node_modules/eas-cli/node_modules/@expo/apple-utils/build/index.js:1:913824)
    at async ApiKey.downloadAsync (/Users/me/.config/yarn/global/node_modules/eas-cli/node_modules/@expo/apple-utils/build/index.js:1:925069)
    at async createAscApiKeyAsync (/Users/me/.config/yarn/global/node_modules/eas-cli/build/credentials/ios/appstore/ascApiKey.js:66:23)
    at async AppStoreApi.createAscApiKeyAsync (/Users/me/.config/yarn/global/node_modules/eas-cli/build/credentials/ios/appstore/AppStoreApi.js:98:16)
    at async generateAscApiKeyAsync (/Users/me/.config/yarn/global/node_modules/eas-cli/build/credentials/ios/actions/AscApiKeyUtils.js:74:23)
    at async provideOrGenerateAscApiKeyAsync (/Users/me/.config/yarn/global/node_modules/eas-cli/build/credentials/ios/actions/AscApiKeyUtils.js:53:16)
    at async CreateAscApiKey.runAsync (/Users/me/.config/yarn/global/node_modules/eas-cli/build/credentials/ios/actions/CreateAscApiKey.js:15:27)
```

### Reproducible demo or steps to reproduce from a blank project

1. `expo credentials`

**Comment by id3vz** [comment]
facing the same issue

**Comment by quinlanj** [comment]
I just released a fix, should be available in 1.1.1 of `eas-cli`. Definitely post back here/make another issue if it does not solve your problem!

**Comment by quinlanj** [comment]
I just released a fix, should be available in 1.1.1 of `eas-cli`. Definitely post back here/make another issue if it does not solve your problem!
